Packs/Kenna/Integrations/KennaV2/KennaV2.yml

```diff
@@ -441,9 +441,28 @@ script:
     - contextPath: Kenna.ConnectorRunsList.ReopenedVulnerabilities
       description: Amount of re-opened vulnerabilities.
       type: Number
-  dockerimage: demisto/python3:3.10.12.68714
+  - arguments:
+    - name: asset_id
+      required: true
+      description: The ID of the asset to inactivate.
+    - name: notes
+      required: true
+      description: Notes about the asset.
+    description: Inactivates a single asset.
+    name: kenna-inactivate-asset
+  - arguments:
+    - name: external_id
+      description: The external_id of the asset.
+    - name: to_context
+      description: Posibility of get data in context.
+      type: Boolean
+    name: kenna-search-assets-by-external-id
+    description: Search assets by external id.
```

Comment:
Please add `outputs` key. 🙏 

e.g.
if the response from the API looks like this
```json
"assets": [
        {
            "id": 3,
            "notes": "Bla Bla",
```
the `outputs` key in the `yml` file should looks like this
```yml
outputs:
    - contextPath: Kenna.Assets.ID
      description: The asset ID.
      type: Number
    - contextPath: Kenna.Assets.Notes
      description: Notes of current asset.
      type: String